src/components/ui/pagination/pagination.tsx

```diff
@@ -46,43 +48,53 @@ const StyledButton = styled.button`
   }
 `;
 
+const PageText = styled.span`
+  ${TextStyles.toplineR}
+  color: ${({ theme }) => theme.palette.text.topline};
+`;
+
 export const Pagination = ({
   onPreviousClick,
   onNextClick,
   amountOfPages,
   onDropdownSelect,
   currentPage,
+  className,
 }: PaginationProps): JSX.Element => {
   const { t } = useTranslation();
 
   const options = new Array(amountOfPages)
     .fill(null)
     .map((_, index) => (
-      <DropdownOption key={index} value={index.toString()}>{`${t(
+      <DropdownOption key={index} value={(index + 1).toString()}>{`${t(
         'blog.pagination',
       )} ${index + 1}`}</DropdownOption>
     ));
 
   const handleDropdownChange = (selectedOption) => {
     const selectedPage = parseInt(selectedOption);
-    onDropdownSelect(selectedPage);
+    onDropdownSelect?.(selectedPage);
   };
 
   return (
-    <PaginationContainer>
-      <StyledButton onClick={onPreviousClick} disabled={currentPage === 0}>
+    <PaginationContainer className={className}>
+      <StyledButton onClick={onPreviousClick} disabled={currentPage === 1}>
```

Comment:
I think from an accessibility perspective, this should be a native link rather than a button. Instead of a link, you can also use `aria` labels to make the button more accessible. Can you please do some research?

Reply:
You are right, According to https://www.a11ymatters.com/pattern/pagination/, Links are better than buttons and I also added area-labels.

They also suggested to wrap the navigation in a <nav> tag, but they used a simple list as pagination. I would suggest to leave that out, but what are your thoughts on that?

Reply:
Very good! We can leave out the list with the links for now. Let's see how this feature evolves over time, maybe we can implement that later.